<launch>

  <param name="picture_topic" value="/strawberry/faces/face_at_center/image_raw" type="string"/>  
  <node pkg="picture_server" name="picture_server" type="picture_server" output="screen"/>
  
</launch>
